<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Coloring for PCB Routing</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <widget class="QPushButton" name="btnSelect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddVertex">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Vertex</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddEdge">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Edge</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnColorGraph">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Graph</string>
    </property>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
